homeassistant/components/anthropic/manifest.json

```diff
@@ -8,5 +8,6 @@
   "documentation": "https://www.home-assistant.io/integrations/anthropic",
   "integration_type": "service",
   "iot_class": "cloud_polling",
+  "loggers": ["homeassistant.components.conversation"],
```

Comment:
I think it would be better if we searched for a more generic/backend solution.

This is out of scope for an integration to control IMHO.

../Frenck